spec/javascript/components/applications/application_form/PaperApplicationForm.test.js

```diff
@@ -66,6 +67,23 @@ describe('PaperApplicationForm', () => {
         `${errorMsgSel}.error`
       expect(wrapper.find(errorElementsSel).text()).toEqual('Please enter a valid dollar amount.')
     })
+
+    test('should validate language', async () => {
```

Comment:
looks like from above that the thought for these tests is to have the test name be the field name?

So then it's

describe PaperApplicationForm
  describe should validate fields correctly: 
     test Annual income
     _test Language_

not sure if that's the best way to go about it, but I think we should at least make the naming consistent and have it make sense